x-pack/packages/ml/trained_models_utils/src/constants/trained_models.ts

```diff
@@ -57,7 +58,48 @@ export const ELASTIC_MODEL_DEFINITIONS = {
       defaultMessage: 'Elastic Learned Sparse EncodeR v1 (Tech Preview)',
     }),
   },
-} as const;
+  '.elser_model_2': {
```

Comment:
_Temporarily_ setting this to SNAPSHOT will mean the model will be available for download right now. Then SNAPSHOT can be removed on feature freeze day when the final model is uploaded.

```suggestion
  '.elser_model_2_SNAPSHOT': {
```

Reply:
Added in https://github.com/elastic/kibana/pull/167000/commits/fc4d83b14b00d3e547d3017fbb5ee1b1477ce808